### ISSUE TITLE
Title Bubble (as currently demonstrated/implemented) is not keyboard-accessible

### ISSUE THREAD

**Issue by patrickhlauke** [dropbox/scooter#33]
http://dropbox.github.io/scooter/components.html#title-bubble currently only works for mouse users. Keyboard users cannot set focus nor trigger the bubble. General advice should be to only apply bubbles to focusable elements. 

Additionally, it will likely be necessary to tweak the JavaScript relating to tooltips, to ensure that the tooltip itself is accessible / announced when it appears (e.g. using `aria-describedby` on the original trigger element, adding `role="tooltip"`, etc). See http://v4-alpha.getbootstrap.com/components/tooltips/ and how it's implemented there.
